src/main/kotlin/net/ccbluex/liquidbounce/config/AutoShopConfig.kt

```diff
@@ -0,0 +1,94 @@
+package net.ccbluex.liquidbounce.config
+
+import com.google.gson.JsonObject
+import com.google.gson.JsonParser
+import net.ccbluex.liquidbounce.features.module.modules.player.autoshop.ModuleAutoShop
+import net.ccbluex.liquidbounce.features.module.modules.player.autoshop.ShopElement
+import net.ccbluex.liquidbounce.utils.client.logger
+import net.minecraft.item.Item
+import net.minecraft.registry.Registries
+import net.minecraft.util.Identifier
+import java.io.File
+
+object AutoShopConfig {
+    val configFolder = File(
+        ConfigSystem.rootFolder, "autoshop-configs"
+    ).apply {
+        if (!exists()) {
+            mkdir()
+            // TODO: load the default config from LiquidCloud
+        }
+    }
+
+    fun load(configFileName: String = ModuleAutoShop.configName): Boolean {
+        val configFile = File(ConfigSystem.rootFolder, "autoshop-configs/$configFileName.json")
+
+        try {
+            val jsonObject = JsonParser().parse(configFile.bufferedReader()).asJsonObject
+            val items = jsonObject.get("Items").asJsonArray
```

Comment:
I recommend creating a data class and deserealising it instead of doing it by hand.